pkg/ddevapp/config.go

```diff
@@ -408,17 +496,20 @@ func determineAppType(basePath string) (string, error) {
 }
 
 // setSiteSettingsPath determines the location for site's db settings file based on apptype.
-func (c *Config) setSiteSettingsPath(appType string) {
-	settingsFilePath := filepath.Join(c.AppRoot, c.Docroot)
-
+func (c *Config) setSiteSettingsPaths(appType string) {
+	settingsFileBasePath := filepath.Join(c.AppRoot, c.Docroot)
+	var settingsFilePath, localSettingsFilePath string
 	switch appType {
 	case "drupal8":
 		fallthrough
 	case "drupal7":
```

Comment:
(not a blocking comment) Perhaps we should do a map[string]string of settings files? Could maybe let us handle the drush settings file better than we are now too.